.github/workflows/check-catalyst.yaml

```
@@ -478,8 +478,11 @@ jobs:
 
     - name: Install Deps
       run: |
+        sudo apt-get update
+        sudo apt-get install -y python3 python3-pip libomp-dev libasan6 make
+        python3 --version | grep ${{ needs.constants.outputs.primary_python_version }}
+        python3 -m pip install -r requirements.txt
         make frontend
```

I'd be curious to test if this is sufficient as a workaround:
```suggestion
        TODO: remove workaround before merging to main
        pip install pennylane-lightning==0.36
        pip install git+https://github.com/PennyLaneAI/pennylane@v0.36.0-rc0
        make frontend
```